catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/api/EndpointUtility.java

```diff
@@ -0,0 +1,13 @@
+package org.codice.ddf.catalog.ui.api;
+
+import ddf.catalog.federation.FederationException;
+import ddf.catalog.source.SourceUnavailableException;
+import ddf.catalog.source.UnsupportedQueryException;
+import org.codice.ddf.catalog.ui.query.cql.CqlQueryResponse;
+import org.codice.ddf.catalog.ui.query.cql.CqlRequest;
+
+public interface EndpointUtility {
```

Comment:
should this go into the `catalog-ui-search-api` module?  that way integrators can pull the API module in as a dependency instead of having to pull in the entire UI module to use this interface.

Reply:
That was my initial thought, but I would also need interfaces for `CqlQueryResponse` and `CqlRequest` to avoid a circular dependency, should I make interfaces for these as well?

Reply:
imo that might make it out of scope